tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,45 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "add": {
+        "DB_PASSWORD": "{{ 8|random_string }}",
+        "DJANGO_SECRET_KEY": "{{ 20|random_string }}",
+        "EDX_OAUTH2_SECRET": "{{ 8|random_string }}",
```

Comment:
This should also be longer.

Reply:
I'll also set `DJANGO_SECRET_KEY` to 50 characters to follow [Django's default string length when generating a secret key](https://github.com/django/django/blob/main/django/core/management/utils.py#L77).